src/Client/webpack.config.js

```diff
@@ -42,7 +42,8 @@ module.exports = {
       }
     },
     hot: true,
-    inline: true
+    inline: true,
+    historyApiFallback: true,
```

Comment:
what#s that doing?

Reply:
This is for url router, with  this webpack-dev-server will route all 404 urls back to index.html. So we can use html5 history api router in the client. See: https://webpack.js.org/configuration/dev-server/#devserver-historyapifallback